code/controllers/subsystem/air.dm

```diff
@@ -391,7 +391,7 @@ SUBSYSTEM_DEF(air)
 		currentrun.len--
 		EG.breakdown_cooldown++
 		EG.dismantle_cooldown++
-		if(EG.breakdown_cooldown >= EXCITED_GROUP_BREAKDOWN_CYCLES)
+		if(EG.breakdown_cooldown >= EXCITED_GROUP_BREAKDOWN_CYCLES && !(EG.turf_reactions & VOLATILE_REACTION))
 			EG.self_breakdown(poke_turfs = TRUE)
```

Comment:
```suggestion
			EG.self_breakdown(poke_turfs = TRUE)
			EG.dismantle_cooldown = EXCITED_GROUP_BREAKDOWN_CYCLES
```
Otherwise, you risk instantly dismantling in a non stable state.